Fix the "cdi_all" test target

arjantijms opened the pull request:
Signed-off-by: arjantijms <arjan.tijms@gmail.com>

arjantijms committed:
Fix the "cdi_all" test target

Signed-off-by: arjantijms <arjan.tijms@gmail.com>

smillidge commented on appserver/tests/appserv-tests/devtests/cdi/osgi-cdi/simple-wab-with-cdi/WebTest.java:
```diff
@@ -51,8 +50,8 @@ public static void main(String[] args) {
-        try { 
-            invoke();
+        try {
+            // invoke();
```
Why's this one commented out?

arjantijms replied:
That's the OSGi testing that would fail because fighter fish is not there. WABs don't deploy correctly at the moment, but I've also disabled the test at a higher level, so we might uncomment that now.

arjantijms committed:
Processing feedback: uncommented invoke method

Note this does not make a difference at this moment since that test is
being disabled (as long as Fighter Fish is not in GF), but it's simply
not needed to be commented out now.

Signed-off-by: arjantijms <arjan.tijms@gmail.com>
